assembly/assemblyCommon/AssemblyAssembly.cc

```diff
@@ -60,6 +70,7 @@ AssemblyAssembly::AssemblyAssembly(const LStepExpressMotionManager* const motion
   vacuum_pickup_ = config->getValue<int>("Vacuum_PickupTool");
   vacuum_spacer_ = config->getValue<int>("Vacuum_Spacers");
   vacuum_basepl_ = config->getValue<int>("Vacuum_Baseplate");
+  vacuum_stage_ = config->getValue<int>("Vacuum_Stage");
```

Comment:
Can you confirm that all the code added for the fourth vacuum line doesn't harm operation with the different hardware setup with only three vacuum lines used? With only three vacuum lines the config parameter 'Vacuum_Stage' should not be required and running the program must still be possible in case this parameter is missing.